### PR TITLE
PML/UCX/WPOOL: fixed coverity issue

### PR DESCRIPTION
**Pull request by hoopoepg**
- fixed issue reported by coverity

### CODE REVIEW

File: opal/mca/common/ucx/common_ucx_wpool.c [hunk 1]
```diff
@@ -215,11 +215,9 @@ opal_common_ucx_wpool_init(opal_common_ucx_wpool_t *wpool,
...
-    if (NULL != wpool) {
-        OBJ_RELEASE(winfo);
-        OBJ_RELEASE(wpool->dflt_winfo);
-        wpool->dflt_winfo = NULL;
-    }
+    OBJ_RELEASE(winfo);
+    OBJ_RELEASE(wpool->dflt_winfo);
+    wpool->dflt_winfo = NULL;
  err_worker_create:
     ucp_cleanup(wpool->ucp_ctx);
```
Comment (artpol84) [inline opal/mca/common/ucx/common_ucx_wpool.c:222]:
```suggestion
    OBJ_DESTRUCT(&wpool->active_workers);
    OBJ_DESTRUCT(&wpool->active_workers);
    ucp_cleanup(wpool->ucp_ctx);
```

**Review by devreal — APPROVED** [review]

**Review by artpol84 — CHANGES_REQUESTED** [review]

File: opal/mca/common/ucx/common_ucx_wpool.c (hunk 1 above)
Comment (devreal) [reply opal/mca/common/ucx/common_ucx_wpool.c:222]: Typo? (one should be `idle_workers`)

```suggestion
    OBJ_DESTRUCT(&wpool->idle_workers);
    OBJ_DESTRUCT(&wpool->active_workers);
    ucp_cleanup(wpool->ucp_ctx);
```

Comment (hoopoepg) [reply opal/mca/common/ucx/common_ucx_wpool.c:224]: updated PR

**Review by artpol84 — APPROVED** [review]

**Comment by artpol84** [comment]
@hoopoepgneed the Sign-off in the second commit

**Comment by hoopoepg** [comment]
every time forget to sign it :)

**Comment by artpol84** [comment]
@janjust FYI
Need to rebase https://github.com/open-mpi/ompi/pull/8212

**Comment by hoopoepg** [comment]
bot:retest

**Pull request merged**